[OSDOCS#19976]: Added files for 4.13.67 z-stream release notes (#112780)
openshift/openshift-docs: modules/zstream-4-13-67-bug-fixes.adoc

:_mod-docs-content-type: REFERENCE
[id="zstream-4-13-67-bug-fixes_{context}"]
= Fixed issues

[role="_abstract"]

> * Before this update, the `osDiskImage.source.id` property in the user-provisioned infrastructure (UPI) ARM template was incorrectly placed. As a consequence, it lead to installation failures. With this release, the `storageProfile.osDiskImage.source.id` property location in the UPI ARM template is corrected. As a result, the UPI ARM template installation on Azure no longer encounters invalid parameter errors. (link:https://issues.redhat.com/browse/OCPBUGS-81587[OCPBUGS-81587])

Reviewer: We should not mention UPI or IPI in our docs. Remove "(UPI)" reference.

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#user-provisioned-infrastructure

> * Before this update, the `osDiskImage.source.id` property in the user-provisioned infrastructure (UPI) ARM template was incorrectly placed. As a consequence, it lead to installation failures. With this release, the `storageProfile.osDiskImage.source.id` property location in the UPI ARM template is corrected. As a result, the UPI ARM template installation on Azure no longer encounters invalid parameter errors. (link:https://issues.redhat.com/browse/OCPBUGS-81587[OCPBUGS-81587])

Reviewer: ```suggestion
* Before this update, the `osDiskImage.source.id` property in the user-provisioned infrastructure ARM template was incorrectly placed (placed where?). As a consequence, it (what is it?) caused installation failures. With this release, the `storageProfile.osDiskImage.source.id` property location in the user-provisioned infrastructure ARM template is corrected. As a result, the user-provisioned infrastructure ARM template installation on Azure no longer encounters invalid parameter errors. (link:https://issues.redhat.com/browse/OCPBUGS-81587[OCPBUGS-81587])
```

Use an attribute for Azure. See the `common-attributes` file.
